hack/build-local-images.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh -e
+
+imagenames=(docker-registry origin deployer recycler docker-builder                sti-builder                f5-router node openvswitch)
```

Comment:
can you add the new image directories for cluster-capacity and federation?

is there a way we could just have this list pull from the output of `ls` and maybe just ignore from a blacklist `base` and `source`?